File: src/test/java/com/znsio/teswiz/screen/vodqa/DragAndDropScreen.java
package com.znsio.teswiz.screen.vodqa;


public abstract class DragAndDropScreen {

    public static DragAndDropScreen get() {
        return com.znsio.teswiz.screen.ScreenRegistry.getScreen(DragAndDropScreen.class);
    }

    public abstract boolean isMessageVisible();
    public abstract DragAndDropScreen dragAndDropCircleObject();
}

File: src/test/resources/playwright/screens/vodqa/drag-and-drop.screen.ts
import { unsupportedScreenAction } from "../screen-route.ts";

const MESSAGE = "Vodqa drag and drop is not supported on web for WEB_ENGINE=playwright-ts.";

export function isMessageVisible(): object {
  return unsupportedScreenAction(MESSAGE);
}

export function dragAndDropCircleObject(): object {
  return unsupportedScreenAction(MESSAGE);
}
